spec/ConfigChanges/ConfigFile.spec.js

```diff
@@ -19,73 +19,83 @@ var rewire = require('rewire');
+            });
 
-    it('resolveConfigFilePath should return file path', function () {
-        spyOn(configFile, 'getIOSProjectname').and.returnValue('iospath');
-        var configPath = path.join('project_dir', 'iospath', 'config.xml');
-        expect(configFile.resolveConfigFilePath('project_dir', 'ios', 'config.xml')).toBe(configPath);
-    });
+            it('should return file path', function () {
+                var stringsPath = path.join(projectDir, 'res', 'values', 'strings.xml');
+                expect(configFile.resolveConfigFilePath('project_dir', 'android', 'strings.xml')).toBe(stringsPath);
+            });
 
-    it('resolveConfigFilePath should return file path', function () {
-        spyOn(configFile, 'getIOSProjectname').and.returnValue('osxpath');
-        var configPath = path.join('project_dir', 'osxpath', 'config.xml');
-        expect(configFile.resolveConfigFilePath('project_dir', 'osx', 'config.xml')).toBe(configPath);
-    });
+            it('should return file path', function () {
+                spyOn(configFile, 'getIOSProjectname').and.returnValue('iospath');
+                var configPath = path.join('project_dir', 'iospath', 'config.xml');
+                expect(configFile.resolveConfigFilePath('project_dir', 'ios', 'config.xml')).toBe(configPath);
+            });
 
-    it('resolveConfigFilePath should return file path', function () {
-        var file = path.join('res', 'xml');
-        var configPath = path.join('project_dir', 'app', 'src', 'main', file, 'xml');
-        expect(configFile.resolveConfigFilePath('project_dir', 'android', file)).toBe(configPath);
-    });
+            it('should return file path', function () {
+                spyOn(configFile, 'getIOSProjectname').and.returnValue('osxpath');
+                var configPath = path.join('project_dir', 'osxpath', 'config.xml');
+                expect(configFile.resolveConfigFilePath('project_dir', 'osx', 'config.xml')).toBe(configPath);
+            });
+
+            it('should return file path', function () {
+                var file = path.join('res', 'xml');
+                var configPath = path.join('project_dir', 'app', 'src', 'main', file, 'xml');
+                expect(configFile.resolveConfigFilePath('project_dir', 'android', file)).toBe(configPath);
+            });
 
-    it('resolveConfigFilePath should return file path', function () {
-        var file = 'res/xml';
-        var configPath = path.join('project_dir', 'app', 'src', 'main', file, 'xml');
-        expect(configFile.resolveConfigFilePath('project_dir', 'android', file)).toBe(configPath);
+            it('should return file path', function () {
+                var file = 'res/xml';
+                var configPath = path.join('project_dir', 'app', 'src', 'main', file, 'xml');
+                expect(configFile.resolveConfigFilePath('project_dir', 'android', file)).toBe(configPath);
+            });
+        });
```

Comment:
What is the difference except for the `var file` being a string vs string created by `path.join`?

If needed, again change the name to something like

* `should return file path when path is normalized`
* `should return file path when path is not normalized`